aspnetcore/web-api/index/samples/7.x/ApiController/SystemDateTime.cs

```diff
@@ -0,0 +1,9 @@
+﻿public class SystemDateTime : IDateTime
+{
+    public string Now => DateTime.Now.ToString();
+}
+
+public interface IDateTime
+{
+    string Now { get; }
+}
```

Comment:
```suggestion
}

```

GitHub likes a blank line at the end of a file. Otherwise you see

<img width="81" alt="image" src="https://user-images.githubusercontent.com/3605364/180892580-da93cd7d-5544-4c51-a246-0c37a88e3dc5.png">